multicorn_das/__init__.py

```diff
@@ -67,15 +73,14 @@ def __crash_recovery(self, e):
                 # Wait for server to be available again
                 log_to_postgres(f'Server unavailable, retrying...', WARNING)
                 sleep(0.5)
-                attempts -= 1
-
+                self.__create_channel()
```

Comment:
There are cases where in the postgres shell, FDW would never reconnect because it was using a broken channel. Worked when re-opening it.